src/assets/style.css

```diff
@@ -327,16 +327,21 @@ main h2.tip-title {
   font-weight: bold;
 }
 
-.tip img {
+.tip .tip-image {
   grid-column: 2;
   grid-row: 1 / span 3;
   object-fit: cover;
-  width: 100%;
+  width: 100%; 
   height: 100%;
   max-height: 300px;
   border-left: 2px solid var(--alternate-background-color);
 }
 
+.tip .tip-image img{
```

Comment:
Minor: please add a space between `img` and `{`. This is kind of a coding convention.